### ISSUE TITLE
No GUI output

### ISSUE THREAD

**Issue by K4KDR** [jmason86/MinXSS_Beacon_Decoder#23]
After commit 321e855, the following occurs when running minxss_beacon_decoder.py on Ubuntu 16.04 (64-bit):

With **minxss_beacon_decoder.py** listening to Direwolf for KISS output:

![input-tab](https://user-images.githubusercontent.com/19886368/46249032-2aa7af80-c3f0-11e8-91cb-ac484b407fb5.png)

... and a valid MinXSS-1 packet is received & decoded by Direwolf:

![direwolf](https://user-images.githubusercontent.com/19886368/46249035-31362700-c3f0-11e8-9586-f7a407c357ca.png)

... **-no-** telemetry is displayed in the GUI:

![gui](https://user-images.githubusercontent.com/19886368/46249037-34c9ae00-c3f0-11e8-89d5-01a9501d868d.png)

... however, raw hex data **IS** written to the log file:

![log-file](https://user-images.githubusercontent.com/19886368/46249040-3abf8f00-c3f0-11e8-8ec7-da4a1b89975d.png)

... also please note that the KISS connection between minxss_beacon_decoder.py & Direwolf crashes after the reception of a packet.

**Comment by K4KDR** [comment]
Additional:

It also appears that the contents of **input_properties.cfg** get erased when this scenario occurs.

**Comment by jmason86** [comment]
Hm... maybe that pull request actually introduced this error. Do you want to try changing back the 0xdb back to 0xdd?

**Comment by K4KDR** [comment]
Thank you for the reply!

Adjusting the KISS byte exception back to the incorrect value did not help, but after pulling down commit bbec6dc , the GUI displays perfectly again!

![screenshot from 2018-09-30 15-28-30](https://user-images.githubusercontent.com/19886368/46261726-c0613e80-c4c5-11e8-888a-bd96d3a049d9.png)

**Comment by jmason86** [comment]
Hm.. would you mind also posting the debug log/ files for those two commits? 
Also, does running the tests/test_send_tcpip_packets.py still work for you in both commits? 

Thanks for helping me track down the root cause of this issue! I don't have any radio equipment at home so I can't replicate this issue.

**Comment by K4KDR** [comment]
Additional note:  I'm not doing any over-the-air (radio) testing either.  Audio of MinXSS-1 packets are played in Audacity & decoded by Direwolf.  The KISS connection sends the decoded data to your decoder.  Happy to provide more detail if helpful at any point.

**Comment by K4KDR** [comment]
A short video of my test setup may be seen at:

https://www.qsl.net/k/k4kdr//video/k4kdr-minxss-test-demo.mp4

**Comment by jmason86** [comment]
Right, finding which commit worked vs didn’t is helpful. I used that info to do a diff on the code but nothing super obvious struck me as the problem which is why I’m hoping to get a hold of the .log files to see what messages it prints out to see what code is actually being executed vs not. But I don’t have any audio recordings and such to play and run through that path of the code, so I just can’t replicate it. Those logs should help though.

**Comment by K4KDR** [comment]
Glad to provide more info!

First, here is the audio file that I test with in case it might be useful to you:

https://www.qsl.net/k/k4kdr//files/minxss/MinXSS-1--packet-x3.wav

... and I see that I opened this issue on 29-Sep-2018, so here are the LOG file entries from that date:

https://www.qsl.net/k/k4kdr//files/minxss/k4kdr-logfile-2018-09-29.txt

... finally, in looking at the OUTPUT folder from that date, I see that the files produced during the times the app was run on that commit were zero in size:

https://www.qsl.net/k/k4kdr//files/minxss/k4kdr-output-folder-2018-09-29.png

**Comment by jmason86** [comment]
Perfect, thanks! 
Ah, so the screenshot from your original post of the log file was actually from a prior run when things were still working? That would clear things up a bit, if that was the case.

**Comment by K4KDR** [comment]
Sorry if my original post was not clear; the best image in the original post was the GUI with **no** data present... that was the real issue (plus the fact that it crashed the KISS connection after one packet):

![gui](https://user-images.githubusercontent.com/19886368/46420317-c17cb080-c6fd-11e8-8750-7e91e59aa0c3.png)

**Comment by jmason86** [comment]
Yep, that's exactly what I was using. Most of the cleaning I've been doing is just refactoring -> move code to well named functions, make variables with better names, etc. The logic shouldn't be impacted, but apparently in this case it was somehow. That does make the diff kind of hard to sift through since it's highlighting the rearranging more so than any logic. But clearly something did change.. just gotta look a little harder.

**Comment by jmason86** [comment]
Think I found the problem (or at least _a_ problem that definitely would've impacted this). See if 5ad9afdd0c90afba95b4971fd5d49fd0432139cb fixes the issue! 

For future reference this is what I found was wrong: 

It looks like this another carry over from the transition from `pyside` to `pyside2`. When I query whether the checkboxes are checked in `pyside` the syntax was `checkbox.isChecked` (i.e., it was a property) but in `pyside2` it is `checkbox.isChecked()` (i.e., it's a function).

**Comment by K4KDR** [comment]
Commit 5ad9afd works just fine on my system with a test packet!

![screenshot from 2018-10-03 20-51-03](https://user-images.githubusercontent.com/19886368/46447364-4fcd5280-c74f-11e8-8fa2-5f3c687dd085.png)
